process/block/metablock.go

```diff
@@ -944,8 +944,17 @@ func (mp *metaProcessor) createMiniBlocks(
 ) (*block.Body, error) {
 	var miniBlocks block.MiniBlockSlice
 
+	miniBlocks = mp.scheduledTxsExecutionHandler.GetScheduledMBs()
```

Comment:
currently metachain does not support scheduled execution.
Is this returning nil for now?

Reply:
It returns an empty block.MiniBlockSlice, but I have changed now to return nil. Also this is important for shard when there are no miniblocks in body. The return should be nil and not emply slice.